src/main/java/com/wasilewskyy/medical_clinic/model/Patient.java

```diff
@@ -9,12 +10,40 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
+@Entity
+@Table(name="patient")
 public class Patient {
+
+    @Column(name="PATIENT_EMAIL",unique=false, nullable=false)
     private String email;
-    private String password;
-    private String idCardNo;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long idCardNo;
```

Comment:
Nie uzywajmy tego jako od zróbmy, żeby to był string <- to jest informacja o numerze dowodu osobistego